test/helper.rb

```diff
@@ -4,10 +4,14 @@
 # add the root dir to the load path
 $LOAD_PATH.unshift(File.expand_path("../..", __FILE__))
 
+TMP_PATH = File.expand_path("../../tmp", __FILE__)
+
 # require pry for debugging (`binding.pry`)
 require 'pry'
 require 'test/support/factory'
 
+# TODO - rework the tests so this isn't needed, requires removing coupling from
+# the main `Ardb` module
```

Comment:
@jcredding two questions: 1) why should this not be needed? and 2) what coupling exists that is preventing this?